plugin/helpers.go

```diff
@@ -78,6 +80,11 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.18
 	InstallPluginFromURL(downloadURL string, replace bool) (*model.Manifest, error)
+
+	// GetPluginAssetURL builds a URL to the resource from asset directory.
+	// Plugins may expose custom assets via the assets directory,
+	// but must then combine the SiteURL and their plugin ID to build the resulting path.
```

Comment:
This text is helpful in the context of the PR, but I think is confusing here, since it sounds like a requirement on the part of the plugin author. Instead, this helper method simplifies doing that work. Propose:
```suggestion
	// GetPluginAssetURL builds a URL to the given asset in the assets directory.
	// Use this URL to link to assets from the webapp, or for third-party integrations with your plugin.
```

Reply:
@lieut-data Thanks for the suggestion, I will work on it.